### ISSUE TITLE
Middle Click Scrolling

### ISSUE THREAD

**Issue by Namchee** [locomotivemtl/locomotive-scroll#180]
**Is your feature request related to a problem? Please describe.**

Hi,

First of all, thank you for sharing this awesome library!
I would like to request a feature where it's possible to scroll with the middle click mouse scrolling for better accessibility. Currently, it's impossible to do this

**Describe the solution you'd like**
An attribute to enable middle click mouse scrolling.

**Describe alternatives you've considered**

**Additional context**

**Comment by CrazyVideoGamez** [comment]
Yeah. It's really nice when I can just use the middle scroll button to scroll to the bottom/top really quickly, instead of having to manually keep rotating the scroll wheel. Also, I like to use it when I'm reading, and I can just make the middle scroll button automatically scroll really slowly for me, so I don't have to do anything, and I can still read.

**Comment by StevenStavrakis** [comment]
Any progress on this/ known solutions? As far as I am concerned this is necessary. Without it there's little to no point in using locomotive.

**Comment by sgeier** [comment]
Yes, also waiting for this one. I raised a ticket earlier. Many users are excluded or wont be able to navigate the site in their common setup. For example, artists who work with pens, they use their pens to press on the page and then tilt up and down with the pen. I would immediately implement Locomotion if this feature would be supported. I also tried fixing it myself but ran into some difficulties with the overflow elements.

**Comment by alexbogias** [comment]
Same here...

**Comment by AndrewNow** [comment]
Bumping this as well - I noticed that [Mutek's site ](https://montreal.mutek.org/en) appears to have scroll hijacking, but they disabled the locomotive scrollbar, which seems to enable middle mouse scrolling.
